legal-api/src/legal_api/services/filings/validations/dissolution.py

```diff
@@ -167,7 +173,8 @@ def validate_parties_address(filing_json, legal_type) -> Optional[list]:
 def _is_dissolution_party_role(roles: list) -> bool:
     return any(role.get('roleType', '').lower() in
                [PartyRole.RoleTypes.CUSTODIAN.value,
-                PartyRole.RoleTypes.LIQUIDATOR.value] for role in roles)
+                PartyRole.RoleTypes.LIQUIDATOR.value,
+                PartyRole.RoleTypes.PROPRIETOR.value] for role in roles)
```

Comment:
Can you remove this since it not needed anymore and please try to add test case for it https://github.com/bcgov/lear/blob/main/legal-api/tests/unit/services/filings/validations/test_dissolution.py